index.html

```diff
@@ -43,9 +44,9 @@ <h3>Game Server</h3>
     <h2>Contributing</h2>
     <p>The project is no where near complete, it receives frequent updates, is on-going and all contributions are
         welcomed.</p>
-    <p>If you are interested in contributing, please join <a href="https://discord.gg/W4Nk9gt">The MMORPG Project</a>
+    <p>If you are interested in contributing, please join us <a href="https://discord.gg/W4Nk9gt">The MMORPG Project</a>
```

Comment:
Adding "us " is a grammatical error, please remove it or come up with another phrase.